### DISCUSSION TITLE
Any change to integrate other streaming platforms.

### DISCUSSION THREAD

**Discussion by Drallas** [FreeTubeApp/FreeTube#2416] [General]
Hello 

Just wondering if it would be possible to include other streaming platforms like **Odysee**, **Rumble** and **Bitchute** etc?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hello. At this time, we have no plans to support anything outside of YouTube. Please see [this](https://docs.freetubeapp.io/faq/#will-you-consider-adding-support-for-services-outside-of-youtube-twitch-soundcloud-peertube-etc) link for more information. The TLDR is that we don't have the time and resources to support more platforms.
